adafruit_magtag/fakerequests.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+# SPDX-FileCopyrightText: Copyright (c) 2020 Melissa LeBlanc-Williams for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+"""
+`adafruit_magtag.fakerequests`
```

Comment:
Why not factor this out into a separate library? It looks like a copy and paste out of other portal libraries.

Reply:
That's the eventual plan ;)

Reply:
When will it be done? This is our third portal library.

Reply:
I'll get to it soon. I have some stuff to clear off my plate first. You can keep an eye on issue #12.